src/models/secret/test-data.ts

```diff
@@ -13,11 +15,14 @@ export const testData: TestDependencyDataType<SecretModel> = [
   {
     __testDescription: 'Valid full object',
     name: 'name',
+    id: 'id',
```

Comment:
remember to also update testdata for 'Invalid empty object', just add the same fields and data as you did for 'Valid full object'

Reply:
updated